artemis-jms-server/src/main/java/org/apache/activemq/artemis/jms/server/config/impl/ConnectionFactoryConfigurationImpl.java

```diff
@@ -623,6 +636,8 @@ public void decode(final ActiveMQBuffer buffer) {
       deserializationBlackList = BufferHelper.readNullableSimpleStringAsString(buffer);
 
       deserializationWhiteList = BufferHelper.readNullableSimpleStringAsString(buffer);
+
+      enable1xPrefixes = buffer.readableBytes() > 0 ? buffer.readBoolean() : null;
```

Comment:
@clebertsuconic @jbertram Should default to 0, null would throw npe here, as field being set is a promiti5